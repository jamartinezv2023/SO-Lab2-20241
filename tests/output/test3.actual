./test_wish.sh: line 33: ./../bin/wish: No such file or directory
