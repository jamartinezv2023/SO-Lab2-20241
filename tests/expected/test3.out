LICENSE
README.md
README1.md
docs
ejercicios_refuerzo
enunciado
s29.pdf
s30.pdf
tester
